invoices/test_utils_test.go

```diff
@@ -275,7 +275,10 @@ func getCircuitKey(htlcID uint64) invpkg.CircuitKey {
 	}
 }
 
-func newTestInvoice(t *testing.T, preimage lntypes.Preimage,
+// newMPPTestInvoice creates a test invoice with a randomly generated payment
+// address, and the parameters provided (defaulting to an hour if no expiry is
+// set.
+func newMPPTestInvoice(t *testing.T, preimage lntypes.Preimage,
```

Comment:
A payment address is fundamentally a probing defense and doesn't necessarily mean the invoice supports MPP.

Maybe we should leave this as `newTestInvoice` and call the new function something else... `getTestInvoice`?

Reply:
Hm... This function doesn't just add a random pay address.  It also has more parameters to customize the invoice, and it actually includes the encoded invoice.

Maybe this one could be `newTestInvoice` and `newInvoice` could be `newMinimalTestInvoice` or `newSimpleTestInvoice`.

Or... maybe we could wrap the functionality of `newInvoice` into this function instead?

Reply:
> Or... maybe we could wrap the functionality of newInvoice into this function instead?

Thought about this as well but it's pretty nice to have `newInvoice` without parameters when you just want the bare bones invoice. 

Since this function is only used by a quite specific piece of generating code (`generateInvoiceExpiryTestData`), I've moved it next to that code and named it `newInvoiceExpiryTestInvoice` so that it's specific use case is more clear.